test/mocks/Test.js

```diff
@@ -23,4 +23,14 @@ describe('Test', function() {
       test.slow().should.equal(200);
     });
   });
+
+  describe('#withEmptyFile', function() {
+    it('should return the test and set the file to undefined', function() {
+      // when
+      var test = new Test().withUndefinedFile();
+
+      // then
+      should.equal(test.file, undefined);
```

Comment:
no additional dependency - included in chai. You can say `var expect = chai.expect;`